4-css-grid/styles/styles.css

```
@@ -0,0 +1,37 @@
+.A {
+    grid-column: 1 / 3;
+    grid-row: 1;
+    background: black;
```

create a common class for all the blocks and put the common properties in it. 

```suggestion
.block {
    background: black;
    }
```
and then in the html you do: `<div class="block A">A</div>`